Anime-App/Feature/Anime/View/AnimeTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AnimeTableViewCell.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 09/08/2023.
+//
+
+import UIKit
+
+class AnimeTableViewCell: UITableViewCell {
```

Comment:
Can you create a delegate to pass the selected anime back to the view controller i talk about this in this video here. 
https://www.youtube.com/watch?v=sZpTpBb2KB0